frontend/src/css/Sidebar.styl

```diff
@@ -28,8 +28,6 @@ sidebarColorDarken = darken(sidebarColor, 15%)
     transition all sidebar-animate-speed
     width sidebar-width
     height "calc(100vh - %s)" % headerHeight
-    background-image url('../img/applat.png')
```

Comment:
if the image "applat.png" isn't used anymore, remove it from the repository

Reply:
Updated.